docs/source/guide/custom_metric.md

```diff
@@ -105,4 +107,137 @@ After adding your code to Label Studio Enterprise, the following could happen:
 - Your code might fail to deploy if there are errors. If you don't see errors in the Label Studio UI, check your web browser console after you attempt to save the code.
 - Your code might fail to run properly based on the format of your JSON annotations. Export an example annotation from your project in Label Studio JSON format and make sure your function handles the annotation as expected outside of Label Studio. 
 
-If you change the labeling configuration for your project, you might need to update the custom agreement metric code to handle the new format of annotations produced.
\ No newline at end of file
+If you change the labeling configuration for your project, you might need to update the custom agreement metric code to handle the new format of annotations produced.
+
+## Set up permissions for a private cloud custom agreement metric
```

Comment:
decided to put it all on one page so that it was less confusing what to do after you set up all the perms stuff, but let me know if it seems overwhelming